eth_tools/commands.py

```diff
@@ -58,6 +61,24 @@ def fetch_blocks(args: dict, web3: Web3):
             writer.writerow(row)
 
 
+def get_balances(args: dict):
+    """
+    Parses 'transfer' events of an ERC20 contract to compute balances
+    """
+    with open(args['addresses']) as f:
+        addresses = json.load(f)
+    start = args['start_block'] if 'start_block' in args.keys() else None
```

Comment:
Because `start` is not a required cli argument and I don't think there is a need to have a default value of 1. It should just be the minimum block number if no default is given and otherwise use given start number.